c2corg_api/models/book.py

```diff
@@ -0,0 +1,101 @@
+from c2corg_api.models import schema, Base, enums
+from c2corg_api.models.document import (
+    ArchiveDocument, Document, schema_document_locale, schema_attributes)
+from c2corg_api.models.schema_utils import get_update_schema,\
+    restrict_schema, get_create_schema
+from c2corg_api.models.utils import copy_attributes, ArrayOfEnum
+from c2corg_common.fields_book import fields_book
+from colanderalchemy import SQLAlchemySchemaNode
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    SmallInteger,
+    ForeignKey
+    )
+from c2corg_common import document_types
+
+BOOK_TYPE = document_types.BOOK_TYPE
+
+
+class _BookMixin(object):
+    author = Column(String(100))
+    editor = Column(String(100))
+    activities = Column(ArrayOfEnum(enums.activity_type))
+    url = Column(String(255))
+    isbn = Column(String(17))
+    book_types = Column(ArrayOfEnum(enums.book_type))
+    nb_pages = Column(SmallInteger)
+
+    # it is varchar so I cant use - Column(DateTime)
+    publication_date = Column(String(100))
+
+    # (psycopg2.ProgrammingError) column books.lang does not exist
+    # lang = Column(String())
```

Comment:
`langs` should be of type `Column(ARRAY(String(2)))`. See also: http://docs.sqlalchemy.org/en/latest/dialects/postgresql.html#sqlalchemy.dialects.postgresql.ARRAY